web/route.go

```
@@ -0,0 +1,115 @@
+	"net/http"
+	"speedtest/config"
+	"speedtest/database"
+	"speedtest/results"
+
+	"github.com/WJQSERVER-STUDIO/go-utils/logger"
+	"github.com/gin-contrib/cors"
+	"github.com/gin-gonic/gin"
+)
+
+var (
+	logw       = logger.Logw
```

golangci-lint (1.62.2)

var `logw` is unused

(unused)